src/coffee/utils/array-sync.coffee

```diff
@@ -46,8 +46,7 @@ angular.module('uiGmapgoogle-maps').factory 'uiGmaparray-sync', [
           set_at: (index) ->
             return if isSetFromScope #important to avoid cyclic forever change loop watch to map event change and back
             value = mapArray.getAt(index)
-            return  unless value
-            return  if not value.lng or not value.lat
+            return unless value and value.lng and value.lat
```

Comment:
I wouldn't say it is less readable, if your used to coffee this is a goto staple for me know. It eliminates tons of redundant code. 

Also `unless` I normally use only if there is one variable to check for truthyness but your case works cause your using a bunch of ands. Basically unless has unexpected behavior at times where it really means `if !(value and value.lng and value.lat)`. 

Anyway this one liner is not worth arguing about.